web/app/controllers/authenticated/projects/index.ts

```diff
@@ -1,7 +1,12 @@
 import Controller from "@ember/controller";
 import AuthenticatedProjectsIndexRoute from "hermes/routes/authenticated/projects/index";
+import { ProjectStatus } from "hermes/types/project-status";
 import { ModelFrom } from "hermes/types/route-models";
 
 export default class AuthenticatedProjectsController extends Controller {
+  queryParams = ["status"];
+
+  status = ProjectStatus.Active;
```

Comment:
What is this line doing?

Reply:
Ah got it, maybe a comment would help clarify for anyone else besides me who didn't immediately understand this 🙂